hs_err_pid9425.log

```diff
@@ -0,0 +1,691 @@
+#
```

Comment:
This should be git-ignored.  And .DS_Store above

Reply:
👍 Thanks! Got some issues with resolving conflicts so maybe this is where it came from.